client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -107,14 +111,18 @@ def prediction_url(self, value):
             raise ValueError("not a valid prediction_url")
         self._prediction_url = parsed.geturl()
 
+    def batch_prediction_url(self):
```

Comment:
```suggestion
    @property
    def _batch_prediction_url(self):
```

Recommending

1. making it a `@property` to match `prediction_url` (`self.batch_prediction_url` instead of `self.batch_prediction_url()`)
2. marking it as internal with an initial underscore because a user shouldn't be interested in using this URL directly (as opposed to `prediction_url`, which we support cURLing)